aws-redshiftserverless-workgroup/src/main/java/software/amazon/redshiftserverless/workgroup/Translator.java

```diff
@@ -75,6 +76,7 @@ static ResourceModel translateFromReadResponse(final GetWorkgroupResponse awsRes
                 .workgroupName(awsResponse.workgroup().workgroupName())
                 .namespaceName(awsResponse.workgroup().namespaceName())
                 .baseCapacity(awsResponse.workgroup().baseCapacity())
+                .maxCapacity(awsResponse.workgroup().maxCapacity())
```

Comment:
I understand maxCapacity can be used in update-workgroup API
is this information provided in get-workgroup response which is invoked by Read Handler ? If yes, then we shouldn't add MaxCapacity in write Only properties. 

For reference : 
```
writeOnlyProperties
Resource properties that can be specified by the user, but can't be returned by a read or list request. Write-only properties are often used to contain passwords, secrets, or other sensitive data.
```
https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-schema.html

Reply:
basically maxCapacity is exactly same type of parameter as baseCapacity
We have another read only for "/properties/Workgroup/MaxCapacity", When we read it, it is under Workgroup